cmd/gb/generate.go

```diff
@@ -29,6 +29,7 @@ See 'go help generate'.
 	Run: func(ctx *gb.Context, args []string) error {
 		env := cmd.MergeEnv(os.Environ(), map[string]string{
 			"GOPATH": fmt.Sprintf("%s%c%s", ctx.Projectdir(), filepath.ListSeparator, filepath.Join(ctx.Projectdir(), "vendor")),
+			"PATH": fmt.Sprintf("%s%c%s", filepath.Join(ctx.Projectdir(), "bin"), filepath.ListSeparator, os.Environ["PATH"]),
```

Comment:
If `$PROJECT/bin` is at the end of the path, then a non-project installation (of say, a tool like [`msgp`](https://github.com/tinylib/msgp)) would override a project-specific installation. I thought it would go well with the project-centric approach of `gb` to prefer the latter over the former.